packages/flutter/lib/src/widgets/form.dart

```diff
@@ -367,7 +367,9 @@ class FormState extends State<Form> {
       if (!validateOnFocusChange || !field._focusNode.hasFocus) {
         final bool isFieldValid = field.validate();
         hasError = !isFieldValid || hasError;
-        errorMessage += field.errorText ?? '';
+        if (errorMessage.isEmpty) {
+          errorMessage = field.errorText ?? '';
```

Comment:
I don't have access to a real device now, but we should check for native behavior. If the concatenation string was confusing, maybe we should concate with 2 line break in between each message

Reply:
but even with adding the 2 line break, we are still facing the same issue, that the voice over is revealing all the error messages at once, we don't actually want that behavior.
We reveal only the first error, after that the user fills the field and pass to the next one.

Reply:
@chunhtai what do you think ?
This change is really crucial as we're working with a big institute that provides benefits for the blind and visually impaired, and they suggested this accessibility improvement. It's key that our system works well for them. 
And thanks in advance!